### PR TITLE
KCD London Details!

### PR DESCRIPTION
**Pull request by jmickey**
The initial details for KCD London are completed. We now have a date and venue and are ready to get the word out!

### CODE REVIEW

File: content/events/2020-london/sponsor.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Sponsor
+---
+
+Kubernetes Community Days is a community-organized, not-for-profit conference series for practitioners.
+
+Sponsoring Kubernetes Community Days events gives your organization an opportunity to speak with practitioners, managers, and executives from companies of all sizes and industries. Whether your goals include recruiting new talent, opening new markets, or connecting with your existing customers, this event will be the place to have those conversations with experts and leaders in tech innovation.
+
+To enquire about sponsoring Kubernetes Community Days London 2020 get in touch via email at [sponsor@kuberentesdays.london](mailto:sponsor@kubernetesdays.london).
```
Comment (dankohn) [inline content/events/2020-london/sponsor.md]: I would encourage you to have the sponsor pricing listed here when you first publish.

Email has a typo.

Comment (oicheryl) [reply content/events/2020-london/sponsor.md]: I'm still finalizing sponsorship, which might be another few weeks. I'd like to get the venue and date up sooner so we can start promoting the event sooner rather than later.

File: content/events/2020-london/_index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  image: gianluca.png
+  twitter: https://twitter.com/GianArb
+  homepage: https://gianarb.it/
+- name: Mihaela Constantinescu
+  image: mihaela.png
+- name: Suraj Narwade
+  image: suraj.png
+  twitter: https://twitter.com/red_suraj
+  homepage: https://suraj.pro
+organizers_email: organisers@kubernetesdays.london
```
Comment (iennae) [inline content/events/2020-london/_index.md]: Hey folks, 

Can you use the KCD provided email address organizers-london@kubernetescommunitydays.org here? It's important that we have year over year context for potential organizers, sponsors, and attendees. It also makes it easier for sponsors who want to sponsor multiple events to "guess" the correct addresses to use as addresses follow the same format across events. 

We are looking into also providing a organisers-london@kubernetescommunitydays.org alias for you. 

Thanks!

File: content/events/2020-london/contact.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: Contact
+---
+
+If you’d like to contact us by email: organisers@kubernetesdays.london
+
```
Comment (iennae) [inline content/events/2020-london/contact.md]: Can you use the KCD provided email address organizers-london@kubernetescommunitydays.org here?

File: content/events/2020-london/sponsor.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Sponsor
+---
+
+Kubernetes Community Days is a community-organized, not-for-profit conference series for practitioners.
+
+Sponsoring Kubernetes Community Days events gives your organization an opportunity to speak with practitioners, managers, and executives from companies of all sizes and industries. Whether your goals include recruiting new talent, opening new markets, or connecting with your existing customers, this event will be the place to have those conversations with experts and leaders in tech innovation.
+
+To enquire about sponsoring Kubernetes Community Days London 2020 get in touch via email at [sponsor@kubernetesdays.london](mailto:sponsor@kubernetesdays.london).
```
Comment (iennae) [inline content/events/2020-london/sponsor.md]: Can you use the KCD provided email address organizers-london@kubernetescommunitydays.org here? 

We are also looking at creating another alias sponsor-london@kubernetescommunitydays.org as well so this can stay consistent in format. Thank you.

File: layouts/partials/events/person.html [hunk 5]
```diff
@@ -4,7 +4,7 @@
     <div class="card-content">
```
Comment (iennae) [inline layouts/partials/events/person.html]: This is great! Thank you for fixing https://github.com/cncf/kubernetes-community-days/issues/268. Please submit this in a separate PR rather than requesting these fixes within this PR. Thank you!

Comment (gianarb) [reply layouts/partials/events/person.html]: This is already fixed. It needs a rebase

**Comment by gianarb** [comment]
@iennae I made a few changes and I think all your comments are addressed now! Can you have a look again?! Thanks a lot

**Comment by ams0** [comment]
@gianarb consider using a `custom.css` like this [one](https://github.com/cncf/kubernetes-community-days/blob/master/content/events/2020-amsterdam/custom.css) to show a hero banner.

**Comment by iennae** [comment]
@gianarb can I get a +1 that this is ready to merge from a London organizer here?

**Comment by iennae** [comment]
Don't forget to add a profile pic to your Twitter account https://twitter.com/kubedayslondon, I'm ready to merge this with a +1 from a London organizer (other than @jmickey ). Thanks!

File: content/events/2020-london/contact.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+If you’d like to contact us by email: organizers-london@kubernetescommunitydays.org
+
+# Want to be Kept Up-to-date?
+
+Subscribe to the Kubernetes Community Days London mailing list [here](https://mailchi.mp/f5ff97451223/kcdlondon-subscribe)
+
+## Want to Get Involved?
+
+Want to help us build the best Kubernetes conference in London? We're looking for volunteers to help us with organising and day-of logistics. If you think this is something you can help us with, fill out the form [here](https://forms.gle/zKnB6czE25EQkroF8) and we'll be in touch soon!
```
Comment (oicheryl) [inline content/events/2020-london/contact.md]: @gianarb I'd like to remove this section because we have enough volunteers now. But I want to get this PR merged ASAP because it's been 2 weeks already.

**Comment by gianarb** [comment]
@iennae we are ready! can you merge please?
